src/pages/contact.tsx

```diff
@@ -1,4 +1,4 @@
-import React, { useState, useRef, useEffect } from 'react';
+import React, { useState, useRef } from 'react';
```

Comment:
I had to refactor this page because `react-hook-form` package was upgraded and syntax changed